### ISSUE TITLE
Should we archive this repo?

### ISSUE THREAD

**Issue by hober** [w3c/html-extension#40]
Is this still used?

cc @LJWatson

**Comment by marcoscaceres** [comment]
Yes, looks like we should. This is in conflict with HTML itself.

**Comment by marcoscaceres** [comment]
Before we archive it, we need to gut the index.html. We should probably turn off gh-pages for this also.

**Comment by LJWatson** [comment]
I think it can be archived. It was an idea for keeping track of the 
miscellaneous bits of HTML, but it didn't really gain any traction even 
at the time AIRC.

**Comment by marcoscaceres** [comment]
@siusin would you mind setting me up with Admin on this repo? I can take care of it if you'd like.

**Comment by siusin** [comment]
@marcoscaceres you already have admin access to this repo... maybe someone set it up before :)

**Comment by marcoscaceres** [comment]
Sorry I got confused @siusin! Thanks for checking for me.
